src/20_Components/amp-font.html

```diff
@@ -0,0 +1,95 @@
+<!-- 
+  #### Introduction
+
+  Learn how to use the `amp-font` component to trigger and monitor the loading of custom fonts on AMP pages.
+
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://ampbyexample.com/components/amp-font/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-font" src="https://cdn.ampproject.org/v0/amp-font-0.1.js"></script>
+  <!-- 
+    Define some CSS classes so that `amp-font` can add or remove those based on whether a font was loaded or is in error-state. Find out about all the supported css classes [here](https://github.com/ampproject/amphtml/blob/b1607dff9446d332afcdead64b63e8d25e74cff7/extensions/amp-font/amp-font.md)
```

Comment:
"some CSS classes" is very unspecific. Maybe: "Define custom CSS rules to style your document  depending on whether a font was loaded successfully or not."